FreshRSS/FreshRSS
# Hoarder Integration

Changes proposed in this pull request:

- Refactored sharing script to an extension.
- Added Integration with Hoarder API to save the articles. 

How to test the feature manually:

1. Install hoarder latest nightly. 
2. Move xExtension-Hoarder into extensions
3. Set your API key and IP address for your hoarder instance. 
4. Submit the configuration.
5. Refresh FreshRSS, and check if articles are saved as a bookmark in Hoarder.

Pull request checklist:


### Acknowledgments
- Thanks to the [xExtension-readeck-button](https://github.com/Joedmin/xExtension-readeck-button) repository for the code references and insight.
